tests/unit/build_test.py

```diff
@@ -103,3 +121,44 @@ def test_build_container_invalid_container_limits(self):
                 'foo': 'bar'
             })
         )
+
+    def test__set_auth_headers_with_empty_dict_and_auth_configs(self):
```

Comment:
Sorry about that, that was just keeping the name exact with the method's leading underscore.  I'll change it first thing in the morning, rebase, and push it up.